HerPublicWebsite/HerPublicWebsite.csproj

```diff
@@ -39,4 +39,9 @@
       <_ContentIncludedByDefault Remove="Views\Partials\LocalAuthorityMessages\Pending\Default.cshtml" />
     </ItemGroup>
 
+    <ItemGroup>
+      <AdditionalFiles Include="Views\Partials\LocalAuthorityMessages\NoLongerParticipating\Default.cshtml" />
+      <AdditionalFiles Include="Views\Questionnaire\NoLongerParticipating.cshtml" />
+    </ItemGroup>

```

Comment:
is this something we would expect?

Reply:
I don't think so? I've not seen anything like this before, and they weren't required for the previous ticket from what I saw, so I've removed them.